azure_arc_servers_jumpstart/oci/ubuntu/terraform/main.tf

```diff
@@ -0,0 +1,162 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "compartment_ocid" {}
+variable "region" {}
+variable "ssh_public_key" {}
```

Comment:
Suggest putting variable declarations in a standalone variables.tf file. See here for example - https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/aws/ubuntu/terraform/variables.tf

Reply:
@JerryMSFT this is a Terraform best practice. Please implement.